packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -246,4 +247,88 @@ void main() {
   }
 }
 ''';
+
+
+}
+
+class GenL10nLocaleResolutionProject extends GenL10nProject {
+  @override
+  Future<void> setUpIn(Directory dir) {
+    this.dir = dir;
+    writeFile(globals.fs.path.join(dir.path, 'lib', 'l10n', 'app_en_CA.arb'), appEnCa);
+    return super.setUpIn(dir);
+  }
+
+  @override
+  String get main => _main;
+  static const String _main = r'''
+import 'package:flutter/material.dart';
+
+import 'l10n/app_localizations.dart';
+
+class Home extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    try {
+      print('get localizations');
+      final AppLocalizations localizations = AppLocalizations.of(context);
+      print('localizations: $localizations');
```

Comment:
I don't think we want these print statements.

Reply:
They were an accident, I left them in while debugging. Removed!